connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectWorkerIntegrationTest.java

```diff
@@ -238,9 +238,6 @@ public void testBrokerCoordinator() throws Exception {
 
         connect.kafka().stopOnlyKafka();
 
-        connect.assertions().assertExactlyNumWorkersAreUp(NUM_WORKERS,
-                "Group of workers did not remain the same after broker shutdown");
-
```

Comment:
This check doesn't seem necessary. A distributed Connect worker does become unavailable when the underlying Kafka cluster is unavailable, and the existing check (that a request to `GET /` is successful) doesn't really prove much about worker liveness.

Reply:
On reflection, I suppose this check does guarantee that none of the workers died as a result of the Kafka cluster going down. But we get that exact same guarantee later on after bringing Kafka back up and verifying that all the workers are actually alive.